src/app/app-form-validators/validators.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+const isIntegersGreaterThanZero = /^([1-9]\d*)$/;
+const validDatePattern = /^(0?[1-9]|[12][0-9]|3[01])[\/](0?[1-9]|1[012])[\/\-]\d{4}$/;
+
+export function LengthValidator(control: AbstractControl): { [error: string]: boolean } | null {
```

Comment:
In JavaScript/Typescript, we usually name methods and functions using the lower camel case format.

Reply:
Oops.. my bad again.